Corona-Warn-App/src/main/res/layout/include_interoperability.xml

```diff
@@ -0,0 +1,228 @@
+                    app:layout_constraintEnd_toEndOf="parent"
+                    app:layout_constraintStart_toStartOf="parent"
+                    app:layout_constraintTop_toTopOf="parent" />
+
+                <androidx.constraintlayout.widget.ConstraintLayout
+                    android:id="@+id/interoperability_title_container"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:layout_marginTop="@dimen/spacing_small"
+                    app:layout_constraintTop_toBottomOf="@id/interoperability_illustration">
+
+                    <TextView
+                        style="@style/headline4"
+                        android:layout_width="match_parent"
+                        android:layout_height="wrap_content"
+                        android:accessibilityHeading="true"
+                        android:text="@{title}"
+                        android:visibility="@{FormatterHelper.formatVisibility(!isOnboarding)}"
+                        app:layout_constraintBottom_toBottomOf="parent"
+                        app:layout_constraintEnd_toEndOf="parent"
+                        app:layout_constraintStart_toStartOf="parent"
+                        app:layout_constraintTop_toTopOf="parent" />
+
+                    <TextView
+                        style="@style/headline6"
+                        android:layout_width="match_parent"
+                        android:layout_height="wrap_content"
+                        android:accessibilityHeading="true"
+                        android:text="@{title}"
+                        android:visibility="@{FormatterHelper.formatVisibility(isOnboarding)}"
+                        app:layout_constraintBottom_toBottomOf="parent"
+                        app:layout_constraintEnd_toEndOf="parent"
+                        app:layout_constraintStart_toStartOf="parent"
+                        app:layout_constraintTop_toTopOf="parent" />
+                </androidx.constraintlayout.widget.ConstraintLayout>
+
+                <TextView
+                    android:id="@+id/label_interoperability_subtitle"
+                    style="@style/body1"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:layout_marginTop="@dimen/spacing_small"
+                    android:accessibilityHeading="true"
+                    android:text="@{subTitle}"
+                    android:visibility="@{FormatterHelper.formatVisibilityText(subTitle)}"
+                    app:layout_constraintEnd_toEndOf="parent"
+                    app:layout_constraintStart_toStartOf="parent"
+                    app:layout_constraintTop_toBottomOf="@id/interoperability_title_container" />
+
+                <TextView
+                    android:id="@+id/label_interoperability_subtitle2"
```

Comment:
make it more specific as `..._subtitle2` -> what is 2?

Reply:
I was thinking of a name for sub sub title 😄 . What do you think about second_section, third_section?